src/plopp/graphics/figscatter3d.py

```diff
@@ -46,6 +46,19 @@ class FigScatter3d(BaseFig):
         The figure title.
     figsize:
         The width and height of the figure, in pixels.
+    camera:
+        Camera configuration, in the form of a dict. Valid entries are:
+        - ``'position'``: the position of the camera, as a ``sc.vector`` or a list of
+          3 numbers
+        - ``'look_at'``: the point the camera is looking at, as a ``sc.vector`` or a
+          list of 3 numbers
+        - ``'near'``: the distance to the near clipping plane (how close to the camera
+          objects can be before they disappear), as a ``sc.scalar`` or a single number
+        - ``'far'``: the distance to the far clipping plane (how far from the camera
+          objects can be before they disappear), as a ``sc.scalar`` or a single number
+        If values are provided as raw numbers instead of Scipp variables, their unit
+        will be assumed to be the same as the unit of the ``x``, ``y``, and ``z``
+        coordinates.
```

Comment:
Docstring duplicated 3 times. Maybe it is worth defining a simple `dataclass` or function with a single docstring?

Reply:
Yes, that's one of the annoying things about the current implementation (and it also applies to all the other args).
Technically, the `scatter3d` doesn't need to have the args listed, because it just forwards everything to the `FigScatter3d`. However, @jl-wynen and I decided to have them there so that they would show up nicely in the function docs (and also in IDEs).

`FigScatter3d` needs to have them listed, because it needs to distinguish between the args it sends to the `Canvas`, to the `ColorMapper` and the ones it sends to the `PointCloud`.
Finally, the `Canvas` needs to have it because it needs to have it.

I have not managed to come up with an elegant solution to remove this duplication. Would be great if you have an idea.